UPDATING.md

```diff
@@ -24,6 +24,17 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+### Renamed "extra" requirments for cloud providers
+
+Subpackages for specific services have been combined into one variant for
+the cloud provider.
```

Comment:
```suggestion
each cloud provider.
```

This line break here also doesn't show up when rendered (check out https://github.com/apache/airflow/blob/eebea6bf2f8a70596697d3e976cf18bc97d5cbff/UPDATING.md for example) so this is a bit hard to read.